receiver/aerospikereceiver/metadata.yaml

```
@@ -220,7 +220,7 @@ metrics:
     enabled: true
     description: Number of points outside the region.
     unit: '{points}'
-    extended_documentation: Total query result points is geo_region_query_points + geo_region_query_falsepos. Aerospike metric geo_regio_query_falspos.
+    extended_documentation: Total query result points is geo_region_query_points + geo_region_query_falsepos. Aerospike metric geo_regio_query_falsepos.
```

```suggestion
    extended_documentation: Total query result points is geo_region_query_points + geo_region_query_falsepos. Aerospike metric geo_region_query_falsepos.
```
Had to look this one up, [metric reference](https://aerospike.com/docs/reference/metrics?prd=aerospike&ctx=all&ver=7&sev=all&q=geo_region_query_falsepos). Running `make generate` will fix the `documentation.md` file with the same typo.